travel/src/main/resources/db/migration/V3__dbscheme.sql

```diff
@@ -0,0 +1,81 @@
+CREATE TABLE IF NOT EXISTS `exchange_rates` (
+    `id` BIGINT NOT NULL AUTO_INCREMENT,
+    `code` VARCHAR(3) DEFAULT NULL,
+    `amount_in_rsd` DECIMAL(13,6) UNSIGNED DEFAULT NULL,
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `UK_code` (`code`)
+    );
+
+CREATE TABLE IF NOT EXISTS `travel_expense_forms` (
+    `id` VARCHAR(36) NOT NULL,
+    `business_trip_end_date` TIMESTAMP NOT NULL,
```

Comment:
do we need this field in this table?

Reply:
Since we have that same field in 'travel_authorization_forms' table I agree that it's not needed to store same information in two places.
Thank you for pointing it out, I will remove it.

Reply:
Do we want to store information if user extends their business trip duration in another field or are we updating initially set business_trip_end_date?

Reply:
I would say we should update initial trip_end_date in that specific scenario

Reply:
Okay, thank you.